fastify.js

```diff
@@ -40,7 +40,8 @@ function build (options) {
   const map = new Map()
 
   // logger utils
-  const genReqId = loggerUtils.reqIdGenFactory()
+  const customGenReqId = options.logger ? options.logger.genReqId : null
+  const genReqId = loggerUtils.reqIdGenFactory(customGenReqId)
```

Comment:
IMHO passing the function to return it is useless
```js
const genReqId = (options.logger ? options.logger.genReqId : null) || loggerUtils.reqIdGenFactory()
```
So the `loggerUtils.reqIdGenFactory` is without params